### ISSUE TITLE
Training time of SwinIR; Impact of learning rate (fix the lr to 1e-5 for x4 fine-tuning is slightly better)

### ISSUE THREAD

**Issue by shengkelong** [JingyunLiang/SwinIR#3]
In my opinion, the transformer costs much  memory. And the paper pointed out that although swinir has fewer parameters, its speed is much slower than RCAN. So I'm curious about the cost of training, thank you.

**Comment by JingyunLiang** [comment]
Experiments are conducted on a machine with 8 Nvidia 2080 Ti GPUs. We use batch_size=32 and less total iterations to save time.

`classical_sr_x2` (trained on DIV2K, patch size = 48x48) takes about **1.75 days** to train for 500K iterations.

`classical_sr_x4` (trained on DIV2K, patch size = 48x48) takes about **0.95 days** to train for 250K iterations. Note that  we fine-tune x3/x4/8 models from x2 and halve the learning rate and total training iterations, for the benefit of reducing training time.

**Comment by shengkelong** [comment]
Thanks, so does it mean that it is difficult  to train on a single card because of the memory even use 16 batchsize?

**Comment by JingyunLiang** [comment]
Feel free to open it if you have more questions.
